### PR TITLE
feat(language): add brainrot (@someStranger8)

### PR DESCRIPTION
**Pull request by someStranger8**
### Description

Added a new brain rot language with words like rizz, cap, glizzy, sigma, etc.

### Checks

- [X] Adding a language or a theme?
  - [X] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
- [X] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [X] Make sure to include your GitHub username prefixed with @ inside parentheses at the end of the PR title.

### CODE REVIEW

**Comment by Miodec** [comment]
Thanks, i hate it. Can you move it to the "other" group?

**Comment by pikapower9080** [comment]
This is what open source is all about.

**Comment by someStranger8** [comment]
> Thanks, i hate it. Can you move it to the "other" group?

Sorry, but WDYM by other group?

**Comment by notTamion** [comment]
> Sorry, but WDYM by other group?

instead of creating a new group with the language just add the language to the group called other

**Comment by someStranger8** [comment]
> > Sorry, but WDYM by other group?
> 
> 
> 
> instead of creating a new group with the language just add the language to the group called other

Mb, will fix

**Comment by slammon** [comment]
chat is this rizz?

**Comment by Miodec** [comment]
Please remove some of the more nsfw phrases.

File: frontend/static/languages/brainrot.json [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    "zesty",
+    "poggers",
+    "quandale",
+    "morbin",
+    "sisyphus",
+    "t-pose",
+    "ratio",
+    "delulu",
+    "mew",
+    "looksmaxing",
```
Comment (dsf3449) [inline frontend/static/languages/brainrot.json:35]: The correct spelling is `looksmaxxing`, see this wikipedia article: https://en.wikipedia.org/wiki/Looksmaxxing

Comment (someStranger8) [reply frontend/static/languages/brainrot.json]: fixed

**Comment by someStranger8** [comment]
> Please remove some of the more nsfw phrases.

got rid of munt, goon, edge, etc.

**Comment by someStranger8** [comment]
Can someone review this please?

**Comment by Miodec** [comment]
Sorry, but after some more thought, I kinda dont want to add this to the main site.

**Pull request closed without merging**